Unable to send comments

```
backend.js:5 POST https://raendom-backend.z52da5wt.xyz/comment/api/ 400
util.js:20 Error: Request failed with 400
    at util.js:17
```

To reproduce try to leave a comment on this [tip](https://stage.youdonotneedacapetobeahero.com/tip/1848)
```
util.js:20 Error: Request failed with 500
    at util.js:17
```

![image](https://user-images.githubusercontent.com/9008074/92191646-a4cbf300-eea7-11ea-9b6f-d8a909334b62.png)

--- a/src/views/TipsAndComments.vue
+++ b/src/views/TipsAndComments.vue
@@ -19,7 +19,7 @@
       </p>
       <SendComment
         :tip-id="tipId"
-        :parent-id="id"
+        v-bind="id && { parentId: id }"
       />
     </div>
     <div class="comments__section">

--- a/src/views/TipsList.vue
+++ b/src/views/TipsList.vue
@@ -96,7 +96,6 @@ export default {
     @include smallest {
       background-color: $background_color;
       border-bottom: 0.075rem solid $search_nav_border_color;
-      margin: 0 -1rem; // Would be better to reorganize layout somehow
     }
 
     .button-plain {

--- a/src/views/UserProfile.vue
+++ b/src/views/UserProfile.vue
@@ -451,7 +451,7 @@ input[type="file"] {
       width: 100%;
     }
 
-    &.tip-input img {
+    &.tip-input ::v-deep .button img {
       height: 100%;
     }
 

--- a/src/views/UserProfile.vue
+++ b/src/views/UserProfile.vue
@@ -180,10 +180,7 @@
           :key="idx"
           :class="divClass === 'tip_stats' ? 'tips_stats_block' : 'stat_block'"
         >
-          <span
-            v-if="stat.value || stat.image"
-            class="stat-value"
-          >
+          <span class="stat-value">
             {{ stat.value }}
           </span>
           <span class="stat-title">

--- a/src/views/UserProfile.vue
+++ b/src/views/UserProfile.vue
@@ -85,9 +85,7 @@
             </button>
           </template>
         </div>
-        <div
-          class="profile__info"
-        >
+        <div class="profile__info">
           <a
             class="profile__username"
             target="_blank"
@@ -105,14 +103,10 @@
             :title="balance"
           >
             <span>{{ $t('Balance') }}</span>
-            <AeAmountFiat
-              :amount="balance"
-            />
+            <AeAmountFiat :amount="balance" />
           </div>
           <div class="profile__row">
-            <div
-              class="location"
-            >
+            <div class="location">
               <img
                 v-if="profile.location.length || currentAddress === address"
                 src="../assets/location.svg"
@@ -232,7 +226,6 @@ export default {
   data() {
     return {
       maxLength: 250,
-      error: false,
       userStats: null,
       editMode: false,
       userCommentCount: 0,

